README.md

```diff
@@ -253,6 +253,15 @@ Inside the Account folder of the Views folder, create a new View named **TODO: n
 <h3>@ViewBag.Message</h3>
 ```
 
+### [![alt text](https://cloud.githubusercontent.com/assets/328367/17298941/0cd29600-5804-11e6-950c-4542416776bf.png)](https://github.com/nexmo-community/nexmo-verify-2fa-dotnet-example/commit/8501c9884faa97139dc4b30f2f41b4c448cd2641) Update Login View
+
+Inside the Account folder of the Views folder, edit the Login and VerifyCode views. In both files, delete the <div> containing the 'Remember Me' checkbox. This will restrict the user from bypassing 2FA verification.  Also, delete the corresponding variable in each of the view models ('SendCodeViewModel' and 'VerifyCodeViewModel').
```

Comment:
This probably shouldn't be in this PR.

Code should be surrounded by backticks.